src/main/java/org/broadinstitute/consent/http/models/darsummary/DARModalDetailsDTO.java

```diff
@@ -136,42 +126,42 @@ public boolean isSensitivePopulation() {
         return psList;
     }
 
-    private List<String> generateDiseasesSummary(Document darDocument) {
-        List<Map<String, String>> ontologies = (List<Map<String, String>>) darDocument.get(DarConstants.ONTOLOGIES);
+    private List<String> generateDiseasesSummary(DataAccessRequest dar) {
+        List<OntologyEntry> ontologies =  dar.data.getOntologies();
```

Comment:
Null check for `data`

Reply:
Additionally, always use the getter on class fields (here and throughout):
```suggestion
        List<OntologyEntry> ontologies =  dar.getData().getOntologies();
```